.github/workflows/docker-build.yml

```diff
@@ -42,25 +54,24 @@ jobs:
             type=ref,event=branch
             type=ref,event=tag
             type=ref,event=pr
-            type=raw,value=3.12-latest
+            type=raw,value=3.13-latest,enable=${{ github.ref == 'refs/heads/master' }}
 
+      # Authenticate with Docker Hub
       - name: Login to DockerHub
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           username: ${{ secrets.DOCKER_HUB_USERNAME }}
           password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
 
+      # Build + push (with cache)
       - name: Build and push
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v5
         with:
-          # Pull forces Docker to check the network for the
-          # latest version of base images.
           pull: true
-          # Push image only when on master.
-          push: ${{ github.ref == 'refs/heads/master' }}
-          # Use Docker Meta to set the tags & OCI Image Format labels.
+          push: true
+          no-cache: true
```

Comment:
I have enabled `no-cache` because we are not going to build this image frequently anyway.